minio/sse.py

```
@@ -76,7 +76,7 @@ def copy_headers(self) -> dict[str, str]:
 class SseKMS(Sse):
     """Server-side encryption - KMS type."""
 
-    def __init__(self, key: str, context: dict[str, Any]):
+    def __init__(self, key: str, context: Optional[dict[str, Any]] = None):
```

Ruff (0.15.18)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)